free-2.md

```diff
@@ -1,44 +1,98 @@
 # Free project 2
 
-
 ## The user and a language
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
-
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+This idea, like Binary Star, is another project I have previously started
+called "ImageOrganizer" (although I will probably change this name). It is
+designed as a simple DSL for organizing large quantities of images based on
+user defined categories. This DSL will help computer users, like myself,
+who generally save a lot of pictures to eventually need to attempt to sort
+them. Although in the past I've designed a simple GUI with buttons
+corresponding to predefined folders, I have always thought it would
+be interesting to build a simple command line language that can be interacted
+with by a customized GUI (similar to how Git GUIs interact with Git). Thus
+the command line tools will allow for simple workflow, classification,
+taging, macros, etc. Although many well defined tools already exist 
+for organizing pictures (even Facebook provides features like this), by
+building a simple scripting language like this, I can design different GUIs
+to be used locally on my computer or remotely on my Android cell phone.
+This means ideally you can organize your local files with a slick
+consistent underlying system with a GUI customized to the current
+enviorment.
```

Comment:
Sounds awesome. As someone with lots and lots of photos I would totally use something like this